packages/compiler/core/parser.ts

```diff
@@ -342,15 +342,15 @@ export function parse(code: string | SourceFile, options: ParseOptions = {}): Ca
 
       if (isBlocklessNamespace(item)) {
         if (seenBlocklessNs) {
-          error({ code: "multiple-blockless-namespace" });
+          error({ code: "multiple-blockless-namespace", target: item });
```

Comment:
should we make the target required here

Reply:
I don't think so. It ends up being pretty handy for target to be current token by default. Most of the time the parser is complaining about the token it's now looking at. But if we have a lot more of these, I could change my mind.

Reply:
ah I see, that why it was showing the next line